bin/wasm_plugins.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Copyright 2019 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Istio Wasm Plugins Script download Stats and Metadata Exchange Plugins
+# and stores them in $OUTDIR/wasm directory.
+
+ROOTDIR=$(cd "$(dirname "$0")"/..; pwd)
```

Comment:
```suggestion
ROOTDIR=$(cd "$(dirname "$0")"/.. && pwd)
```

Or:
```suggestion
ROOTDIR=$(readlink -f "$(dirname "$0")/..")
```

Reply:
Removed ROOTDIR as it was not getting used.